src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -293,4 +303,41 @@ private static WebACAuthorization createAuthorizationFromMap(final Map<String, L
             return Optional.empty();
         }
     }
+
+    public List<WebACAuthorization> getDefaultAuthorizations() {
+        final Map<String, List<String>> aclTriples = new HashMap<>();
+        final List<WebACAuthorization> authorizations = new ArrayList<>();
+
+        getDefaultAcl().listStatements().forEachRemaining(x -> {
+            if (isAclPredicate.test(x.getPredicate())) {
+                final Triple t = x.asTriple();
+                aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
+                if (t.getObject().isURI()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(t.getObject().getURI());
+                } else if (t.getObject().isLiteral()) {
+                    aclTriples.get(t.getPredicate().getURI()).add(
+                        t.getObject().getLiteralValue().toString());
+                }
+            }
+        });
+
+        authorizations.add(createAuthorizationFromMap(aclTriples));
+
+        return authorizations;
+    }
+
+    private static Model getDefaultAcl() {
+        final String rootAcl = System.getProperty(ROOT_ACL_PROPERTY);
+
+        if (rootAcl != null && new File(rootAcl).isFile()) {
+            LOGGER.debug("Getting Root ACL from file: {}", rootAcl);
+            return RDFDataMgr.loadModel(rootAcl);
```

Comment:
I do not see any exception handling here or in the call structure. What if the file exists but is malformed RDF? Could you add a test for that?